quipucords/quipucords/settings.py

```diff
@@ -201,9 +201,18 @@
 LOGGING_HANDLERS = os.getenv('DJANGO_LOG_HANDLERS', 'console').split(',')
 VERBOSE_FORMATTING = '%(levelname)s %(asctime)s %(module)s ' \
     '%(process)d %(thread)d %(message)s'
-DEFAULT_LOG_FILE = os.path.join(BASE_DIR, 'app.log')
+
+LOG_DIRECTORY = os.getenv('LOG_DIRECTORY', BASE_DIR)
+DEFAULT_LOG_FILE = os.path.join(LOG_DIRECTORY, 'app.log')
 LOGGING_FILE = os.getenv('DJANGO_LOG_FILE', DEFAULT_LOG_FILE)
 
+DEFAULT_SCAN_DATA_LOG = os.path.join(LOG_DIRECTORY, 'scan_data_log')
```

Comment:
Will the log file end with the '.log' suffix? and 'scan_data_log' is a base name for the rotating filename?

Reply:
Also maybe end in `_FILE` to stay consistent with `DEFAULT_LOG_FILE`.

Reply:
Just renamed to end in `.log` instead of `_log`. The `RotatingFileHandler` will make files `scan_data.log.1`, `scan_data.log.2`, etc. for old logs.

Reply:
And changed to end in `_FILE`.